lib/y2_installbase.pm

```diff
@@ -207,6 +207,9 @@ sub process_patterns {
             $self->select_all_patterns_by_menu();
             $self->deselect_pattern() if get_var('EXCLUDE_PATTERNS');
         }
+        elsif (get_var('PATTERNS', '') !~ /default|all/) {
```

Comment:
we should create a new test module, this logic here is not clear why and this is used extensively, we need to narrow where we place this IMO. Besides, it is not transparent that the patterns needs to be visible.
A new test modules `select_visible_patterns` with `$self->process_visible_patterns();` would be much more clear IMO.

Reply:
Additionally we need to apply this change to one test suite at least, but it would be ideal to apply to all the test suites in YaST group that needs this (I think they are are few afir, but could be handle in differen ticket).

Reply:
Updated